doc/filters/markdown_to_html.rst

```diff
@@ -60,3 +62,6 @@ You can also use the filter on an included file:
                 }
             }
         });
+       
+    Afterwards you need to install a markdown library of your choice. Some of them are
+    mentioned in ``require-dev`` section of ``twig/markdown-extra``.
```

Comment:
[...] in the ``require-dev`` section of the ``twig/markdown-extra`` package.

Reply:
Done